### ISSUE TITLE
Please add Edit button to all tasks

### ISSUE THREAD

**Issue by SVilgelm** [iulian-radu-at/vscode-tasks-sidebar#1]
Hi, thank you for your extension, works great.
The only flaw if have with this extension is the missing Edit button, that will let me to quickly open the tasks.json file and modify it

**Comment by iulian-radu-at** [comment]
Hi @SVilgelm,

Thanks for the suggestion.

Where do you expect to see this button and how should it look?

Cheers,
Iulian

**Comment by SVilgelm** [comment]
Something like this, but an edit button :)

![image](https://github.com/iulian-radu-at/vscode-tasks-sidebar/assets/523825/a45ff402-a933-4e07-af27-d5dc072d9f9b)

It would be great if that button will open the selected task in tasks.json

**Comment by kanlukasz** [comment]
Yes, I just tested this extension and immediately ran out of this option.
My suggestion is to add a right-click context menu with the "Run" and "Edit" options.
Something like this:

![image](https://github.com/iulianraduat/vscode-tasks-sidebar/assets/30685349/3b37e173-7f51-4157-9431-c43e5fa74789)

**Comment by iulianraduat** [comment]
Hi there,

The list of tasks are provided by VSCode using the information provided by different providers. And it does not present to me from which configuration files they are read.
I will try to "fake" it and to find possible source files for them so that you can edit them.

Cheers,
Iulian
